archive/problems.json

```diff
@@ -2319,7 +2312,7 @@
   {
     "code": "most-visited-sector-in-a-circular-track",
     "problemNumber": 1560,
-    "name": "Most Visited Sector in a Circular Track",
+    "name": "Most Visited Sector in  a Circular Track",
```

Comment:
Note to self to add back the whitespace collapsing -- seems like there is a LeetCode typo here.